src/components/Header.vue

```diff
@@ -121,8 +119,18 @@ i {
   width: 100%;
   top: 0;
   z-index: 2;
+  transition: 0.4s ease-in;
+  .scrolledHeigth {
+    @media screen and (min-width: $breakpoint-desktop) {
+      min-height: 10vh;
+      transition: 0.4s ease-out;
+    }
+  }
+}
+.scrolledOpacity {
```

Comment:
I think you could add a hover selector to return the opacity to 1. It will add a nice effect.

Reply:
Done